### PR TITLE
Fix signature-request for new Storybook format

### PR DESCRIPTION
**Pull request by EtDu**
Updating `SignatureRequest` story:
- Add js doc comments to proptypes to allow ArgsTable to show up
- Add README.MDX
- Add controls for interaction of component

**Manual testing steps**
- Go to latest CI build of storybook
- Search "SignatureRequest"
- Use Controls to interact with component
- Check docs page to see Props table

**Images**

Before

Screen Shot 2021-12-01 at 12 09 48 PM

After

Screen Shot 2021-12-01 at 12 10 09 PM

screencapture-localhost-6006-2021-12-01-12_10_45

### CODE REVIEW

**Review by darkwing — DISMISSED** [review]

File: ui/components/app/signature-request/README.mdx [hunk 1]
```diff
@@ -0,0 +1,15 @@
+import { Story, Canvas, ArgsTable } from '@storybook/addon-docs';
+
+import SignatureRequest from '.';
+
+# Signature Request
+
+Buttons communicate actions that users can take.
```
Comment (georgewrmarshall) [inline ui/components/app/signature-request/README.mdx:7]: Needs updating to what the component does.

**Review by georgewrmarshall — COMMENTED** [review]
Need to update description

**Review by georgewrmarshall — DISMISSED** [review]
LGTM

**Review by georgewrmarshall — APPROVED** [review]
lgtm

**Review by adonesky1 — APPROVED** [review]
LGTM

**Pull request merged**